src/ddscxx/src/org/eclipse/cyclonedds/sub/qos/DataReaderQosDelegate.cpp

```diff
@@ -125,6 +125,13 @@ DataReaderQosDelegate::policy(const dds::core::policy::ReaderDataLifecycle& life
     lifecycle_ = lifecycle;
 }
 
+void
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro. (BTW: I see that in this particular case the embedding OMG_DDS_OWNERSHIP_SUPPORT macro was also missing for the "ownership" policy).

Reply:
Will do.